src/Microsoft.AspNetCore.Mvc.Core/MvcOptions.cs

```diff
@@ -65,6 +65,8 @@ public MvcOptions()
             };
         }
 
+        public bool EnableGlobalRouting { get; set; }
```

Comment:
This also needs a backing field and doc comments (most of which can be copy-paste. You might be the lucky winner who gets to add a `2.2 setting to the compatiblity switch enum`.

Reply:
See other compat switches, and the novel I wrote elsewhere (on one of the interfaces or something).